ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/GODictionaryRenderer/GODictionaryRenderer.cpp

```diff
@@ -94,6 +94,8 @@ void GODictionaryRenderer::componentComplete()
   m_graphicsOverlay->setRenderer(renderer);
   //! [Apply Dictionary Renderer Graphics Overlay Cpp]
 
+  Q_UNUSED(specType);
```

Comment:
Not important, but there is a unused variable in snippet. This is just to remove the warning.